bnd.bnd

```diff
@@ -0,0 +1,29 @@
+-noextraheader:true
```

Comment:
no need to overwrite the default config: https://github.com/apache/sling-parent/blob/35ff6b88ff7e69a9da0f94be1997b7a72450282e/sling-bundle-parent/pom.xml#L66

Reply:
+1, done now